auth-external-api/src/main/java/uk/gov/di/authentication/external/lambda/UserInfoHandler.java

```diff
@@ -186,6 +186,8 @@ public APIGatewayProxyResponseEvent userInfoRequestHandler(APIGatewayProxyReques
                     accessToken.getValue());
         }
 
+        sessionService.save(userSession.setNewAccount(Session.AccountState.EXISTING));
+        LOG.info("Set account state to EXISTING");
```

Comment:
Is this logging useful?

Reply:
It was useful for testing, doesn't particularly tell you anything useful outside of "the code did its job" which is expected. Will remove 🙂